hathor/cli/run_node.py

```diff
@@ -99,6 +99,8 @@ def create_parser(self) -> ArgumentParser:
                             help='Disable support for running sync-v1. DO NOT ENABLE, IT WILL BREAK.')
         parser.add_argument('--x-localhost-only', action='store_true', help='Only connect to peers on localhost')
         parser.add_argument('--x-rocksdb-indexes', action='store_true', help='Use RocksDB indexes (currently opt-in)')
+        parser.add_argument('--enable-event-queue', action='store_true', help='Enable event queue mechanism')
```

Comment:
Should we prefix with `--x-` (e.g., --x-enable-event-queue) while it's under development? It might be misleading for sysadmins to see these new flags available.

We might also exit the process with an error message saying that this feature is not available yet.

Reply:
I agree. I've sent a new commit with this change